src/orquestra/sdk/_shared/__init__.py

```diff
@@ -0,0 +1,35 @@
+from ._graphs import iter_invocations_topologically
+from ._logs import (
+    LogAccumulator,
+    LogOutput,
+    LogReader,
+    LogStreamType,
+    WorkflowLogs,
+    is_env_setup,
+    is_worker,
+)
```

Comment:
WDYT about leaving these inside a to-be-public `_shared.log` package? The names `is_env_setup` and `is_worker` make much more sense when they're "namespaced" by the `logs` module.

Reply:
Good idea, Ill just remove it from top-level init in shared